projects/app/src/util/date.ts

```
@@ -1,14 +1,55 @@
+  // Find the most significant time unit for simplicity
+  const timeUnits = [
+    `years`,
+    `months`,
+    `weeks`,
+    `days`,
+    `hours`,
+    `minutes`,
+  ] as const;
+  const mostSignificantUnit = timeUnits.find((unit) => {
+    const value = duration[unit];
+    return typeof value === `number` && value > 0;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)